README.md

```diff
@@ -5,13 +5,13 @@
 <!-- default badges end -->
 # HtmlEditor for DevExtreme - How to implement mail merge with variables
 
-The HtmlEditor component supports [variables](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/variables/). This repository shows how to implement mail merge with variables.
+The HtmlEditor component supports [variables](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/variables/). This example demonstrates how to implement mail merge with variables.
 
-The application contains HtmlEditor and two TextBoxes. You can type a value in the TextBoxes which is used to merge with the variable.
+The application contains HtmlEditor and two TextBoxes. Type a value in the TextBoxes to see it merged with HtmlEditor text.
```

Comment:
`Type a value in the TextBoxes to see it merged with HtmlEditor text.`

1. Is "TextBox" the actual control name? Or are you talking about "text boxes" in a general sense?

2. Are you only typing one value (the same value) in both TextBoxes? Or are you telling the reader to type a separate value in each TextBox:

`Type a value in each TextBox to see it merged with HtmlEditor text.`

Reply:
These are different values. For example, the first TextBox is for the name insert, the other one is for surname insert.

Reply:
Okay, since they are different values, can we update this line to the following?

```suggestion
The application contains HtmlEditor and two TextBoxes. Type values in the TextBoxes to see them merged with HtmlEditor text.
```